src/pages/who-we-are/index.tsx

```diff
@@ -35,8 +37,66 @@ const EndSeparator = styled.div`
         border-bottom: 40px solid transparent;
     }
 `
-
 const AboutUs = () => {
+    const target1 = '#simti'
+    const options = {
+        root: null,
+        rootMargin: '0px',
+        threshold: 0.7,
+    }
+
+    const lazy_components: ReactNode = (
+        <>
+            {/* test */}
+            <OurValues
+                fallback={
```

Comment:
Just a suggestion, since fallback is repeating you can create a component for it

```
const Fallback = ({ src, alt }) => (
     <div>
            <img src={src} style={{ width: '95%' }} alt={alt} />
      </div>
)
```